### PR TITLE
Farm home

### PR DESCRIPTION
**Pull request by tiberiusferreira**
Now the products proposed by the farm can be changed. 
There are still some UI changes and Server Side bugs to be ironed out.

capture d ecran 2017-03-11 a 19 22 52

### CODE REVIEW

File: src/routes/FarmHome.vue [hunk 1]
```diff
@@ -43,89 +65,72 @@
...
 <script>
-  let farmid
+  import * as api from '../api/general'
+  import settings from '../config/settings'
+  let farm_id, farm, all_products
   export default {
     /* global localStorage:true */
     data () {
+      // get user info
       this.$http.get('http://ec2-52-56-114-123.eu-west-2.compute.amazonaws.com/auth/user', {
```
Comment (LoicBer) [inline src/routes/FarmHome.vue]: Could you replace these hard coded links and use the settings file ?

**Review by LoicBer — COMMENTED** [review]
It looks good. Now it's simpler to edit the farm information all at once. More features and the code is more clear :smiley: . I would just remove the hard coded links.

File: src/routes/FarmHome.vue (hunk 1 above)
Comment (tiberiusferreira) [reply src/routes/FarmHome.vue]: Oh yeah, totally forgot about it. I'm on it.

Comment (tiberiusferreira) [reply src/routes/FarmHome.vue]: Replaced them. If I missed some link, don't hesitate to call my attention to it :) .

**Review by LoicBer — APPROVED** [review]

**Pull request merged**